Dockerfile

```diff
@@ -22,9 +22,9 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-# Use CentOS7
-FROM centos:7
+FROM inveniosoftware/centos7-python:3.6
 
+ENV XROOTD_VERSION=4.12.2
```

Comment:
Perhaps leave a small note about why we pin to 4.12.2 because of those XRootD  `-v` versions. Just so that it would be obvious for next time.